README.md

```diff
@@ -1,7 +1,7 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Hello Microverse
```

Comment:
- [ ] Kindly fix your project title on the readme. You have a title for Hello Microverse from the first week.

- [ ] Kindly include detailed steps on how to set up the project and run it on the local machine as outlined in the template